README.md

```diff
@@ -11,6 +11,14 @@ This project started as an experiment and even though it was stable enough **it
 
 	var wm = new Ventus.WindowManager();
 	
+### Creating a new window manager and disable expose on right-click
+Set exposeOnRightClick on false to disable expose on right click.
+Default is true (right click shows expose).
+
+	var wm = new Ventus.WindowManager({
+		exposeOnRightClick: true
```

Comment:
I was just going to suggest to change the previous name to something like this. Maybe something shorther? exposeOnMouse? or maybe use a string to specify a trigger, something like exposeShortcut = 'rightclick' | 'a' | 'enter'...